requirements-development.txt

```diff
@@ -0,0 +1,6 @@
+-r requirements-editable.txt
+-r requirements-docs.txt
+-r requirements-testing.txt
+-r requirements-linting.txt
+
+-e .
```

Comment:
What do you think about leaning on requirement files? The job is then to automatically create these files as needed...

Reply:
Happily :)